### PR TITLE
Sub nav

### PR DESCRIPTION
**Pull request by julio8a**
Discussion on #1073 

default nav follows ArcGIS Online styling.
modifier matches Esri.com styling (dark sub-nav)

Needs review and possibly move to flexbox.

### CODE REVIEW

**Comment by macandcheese** [comment]
I think using flex vs. float makes more sense here, will also help with the mobile breakpoint. Do we need some kind of logic to "smartly" break to dropdown? This pattern is really only sustainable to a certain # of items, when combined with fixed breakpoints in CW.

**Comment by macandcheese** [comment]
Weird active style -
Screen Shot 2019-04-19 at 2 33 35 PM

Mobile overlap - 
Screen Shot 2019-04-19 at 2 33 58 PM

File: lib/sass/calcite-web/patterns/_sub-nav.scss [hunk 1]
```diff
@@ -9,25 +9,31 @@
...
+  display: flex;
+  justify-content: flex-start;
 }
 
 @mixin sub-nav-list() {
   width: auto;
-  @include right();
+  display: flex;
+  justify-content: flex-end;
+  flex: 1;
```
Comment (paulcpederson) [inline lib/sass/calcite-web/patterns/_sub-nav.scss]: Did you test this in different browsers? I think IE11 and some other browsers have different default implicit values for the single unit shorthand of the flex property.

File: lib/sass/calcite-web/patterns/_sub-nav.scss [hunk 2]
```diff
@@ -9,25 +9,31 @@
...
 }
 
 @mixin sub-nav-link() {
-  padding: $baseline*.9 $baseline;
-  @include left();
+  padding: $baseline*.86 $baseline;
```
Comment (paulcpederson) [inline lib/sass/calcite-web/patterns/_sub-nav.scss]: magic number?

**Comment by macandcheese** [comment]
Couple more visual issues on smaller screen sizes, we may want to add some functionality to append to an overflow menu

Screen Shot 2019-04-25 at 1 59 23 PM
Screen Shot 2019-04-25 at 1 59 13 PM

File: lib/sass/calcite-web/patterns/_sub-nav.scss (hunk 1 above)
Comment (julio8a) [reply lib/sass/calcite-web/patterns/_sub-nav.scss]: Tested in IE11, Edge, Firefox, Safari, and Chrome. No change.

**Pull request closed without merging**